src/modules/components/member/memberDataListItem/memberDataListItem.tsx

```diff
@@ -0,0 +1,83 @@
+import classNames from 'classnames';
+import { Card, Heading, NumberFormat, Tag, formatterUtils } from '../../../../core';
+import { truncateEthAddress } from '../../../../utils/truncateEthereumAddress';
+import { MemberAvatar, type IMemberAvatarProps } from '../memberAvatar';
+
+export interface IMemberDataListProps extends Omit<IMemberAvatarProps, 'size' | 'responsiveSize'> {
+    /**
+     * Whether the member is a delegate of current user or not.
+     */
+    isDelegate?: boolean;
+    /**
+     * Whether the member card is the current user or not.
+     */
+    isCurrentUser?: boolean;
```

Comment:
We could get this information from the `useAccount` hook, see for instance Fabrice's PR here => https://github.com/aragon/ods/pull/122/files#diff-61347f97c5559aaa9cdc9b7668e51aceb7b2bc5ac7c1d7c7b1f3dabbb127adbaR18

Reply:
Okay. I had a version of this, but I was thinking that it come from the backend with no resolutions happening here. Will add.